Example/Podfile.lock

```diff
@@ -85,9 +84,9 @@ EXTERNAL SOURCES:
   Artsy+UIFonts:
     :git: https://github.com/artsy/Artsy-UIFonts.git
   Emission:
-    :path: ../
+    :path: "../"
   React:
-    :path: ../node_modules/react-native
+    :path: "../node_modules/react-native"
```

Comment:
The system Ruby I use is `2.0.0p648`. We don’t have bundler in Emission, did you think of Eigen?